fossevents/templates/base.html

```diff
@@ -16,14 +16,39 @@
     {% compress css %}
     <link rel="stylesheet" type="text/css" href="{% static 'css/normalize.css' %}" />
     <link rel="stylesheet" type="text/x-scss" href="{% static 'css/main.scss' %}" />
+      <link type="text/css" href="{% static 'css/style.css' %}" rel="stylesheet">
+      <link type="text/css" href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
```

Comment:
If it's possible, let's try things without bootstrap. 

Given the number of UI elements, bootstrap is overkill. It messes up with the semantic markup of html that second issue and hence maintenance in long term.